unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/FriendsController/FriendsController.cs

```diff
@@ -146,11 +145,23 @@ public async UniTask<FriendRequest> RejectFriendshipAsync(string friendRequestId
         public void RemoveFriend(string friendId) =>
             apiBridge.RemoveFriend(friendId);
 
-        public void GetFriends(int limit, int skip) =>
-            apiBridge.GetFriends(limit, skip);
+        public async UniTask<string[]> GetFriendsAsync(int limit, int skip, CancellationToken ct)
```

Comment:
lets rename to `cancellationToken` instead of `ct` for both overloads

Reply:
Done!